packages/dds/sequence/src/sequenceFactory.ts

```diff
@@ -102,6 +104,8 @@ export class SharedObjectSequenceFactory implements IChannelFactory {
             }
             return seg;
         }
+
+        throw new Error(`Unrecognized IJSONObject: '${JSON.stringify(segSpec)}'`);
```

Comment:
I'd worry this risks exposing PII in error messages. I think just the message without stringifying it is probably fine. If this does get hit in the future we can probably figure out why based on recent changes.

Reply:
(here and above)